### DISCUSSION TITLE
[Game Economy] Burn Event 01: The Big Wave

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#3374] [Ideas]
### **`A big wave is approaching our islands! Donate wood to the community centre so that we can build a wall to protect the Plaza!`**

_SUMMARY: I would like to propose a 1 month event that is solely focused on burning resources. In summary, a wave is approaching our islands and we need to cdonate our resources to ensure the Plaza's survival! The donation model can be set up at the Plaza in the square. A leaderboard would be shown with top contributors. Top x donators get a prize along with participation badges/rewards for those who donate a minimum of x resources and drawbacks if the community fails to donate the target amount._

### Rewards & Drawbacks

We could set up individual 'step rewards' (values are placeholders):

Donate 10 wood = Sunflower Collectible Deco
Donate 100 wood = Sunflower Collectible Deco
Donate 1,000 wood = Sunflower Collectible Deco
Donate 10,000 wood = Bumpkin Wearable Deco
Donate 100,000 wood = Utility item
Donate 1,000,000 wood = Strong Utility item

Additionally, we could set up community rewards OR even punishments failing to meet the target... some of the punishments could be less SFL profit for the SFL deliveries for x days, maybe a few NPCs go missing for some time lost in the floods (Hopefully Tywin we won't miss him) or other suggestions.

### Why should this event happen?

- Sets a goal for the community to reach, a sense of togetherness.
- The experienced players who have a tonne of wood saved can finally use it for something. there are players with 100k+ wood sitting idle.
- Market action, players would be trading more wood with each other on the market similar to how the Goblin War stimulated the market
- A secondary goal/event to participate in given that some players are season-fatigued or not participating in the season as much

Open for suggestions and feedback! Would love to see these types of events happen more and more :)
Obviously, each event could ask for different resources! just using this for wood as example as I had thought of a tsunami impacting our land in the past XD

### **EDIT:
Adding after discussions on Discord**

Just to summarise before i go into a meeting and forget about all we talked about:

- Event idea sounds cool, we should have it.

- Event timeline, ~4 weeks? Depends on a few other factors such as how complex it is to maintain/set up

- What do players actually need to do? Donate resources. How and which resources to donate is up for discussion. 

- Set up rewards. Each donation = 1 donation ticket, use donation tickets to buy items/decos etc. at the blacksmith at end of event.

- Other: Set up leaderboard and visual cue of wall being built (maybe something in the middle of plaza) that can tell us how our community goal is going

___

Further discuss which resources to be used for donating, and how the donations will occur along with donation tickkets. Some suggestions:

A) Donate 50 x wood to donation NPC. Each donation = 1 donation ticket. Unlimited donations. 1x donate or 10x donate buttons to facilitate donating large amounts (or other).

B) Allow ANY resource to be used for donating by assigning resources with points based on ingame value to SFL.

C) Each week the type of resource required changes. Week 1 could be 50 wood per donation, week 2 could be 50k sunflowers per donation etc. Important to assign an equal enough value to each resource

D) Similar to C, but instead of 1 resource it is a pair of resources. EG. Donate 50 wood, 1k sunflowers and 10 stones week 1. week 2 donate 30 eggplants, 5 iron etc.
Important to assign an equal enough value to each resource pair.

### **EDIT 2:**

Simplest solution presented
![image](https://github.com/sunflower-land/sunflower-land/assets/108110065/1e792bcb-8ffe-4f67-bf00-7815a19e5387)

**Comment by NycksonV** [comment]
It seems to be a very interesting event but I think we should diversify the resources a little more, besides the amounts of wood in my opinion are a little...too extreme and only benefit the whales in the game

**Reply by TheDcol** [reply]
> besides the amounts of wood in my opinion are a little...too extreme and only benefit the whales in the game

hello! do you mean the 50x wood value I proposed is extreme?

As far as varied resources, I would lean on more towards resources that make a bit more sense in terms of the event. Here we need to build a wooden wall, so wood makes sense.. The next event could be a Locust Infestation and require crops

**Comment by Ario8260** [comment]
I agree! It can be an interesting idea to burn the resources and excitement of the game!

**Comment by Oubia973** [comment]
Nice DCol ! Do it please we need it.

**Comment by coldcola** [comment]
> It seems to be a very interesting event but I think we should diversify the resources a little more, besides the amounts of wood in my opinion are a little...too extreme and only benefit the whales in the game

yes as proposed as a limited time event this very clearly prioritizes players who already have a massive stack of resources at the time, and those who wish to purchase them in that limited time.
why should this not be proposed as a permanent burn if people are holding this much and leading to conversations where burns/sinks are thought to be necessary?

**Reply by TheDcol** [reply]
> why should this not be proposed as a permanent burn if people are holding this much

Thanks for the comment, I think if there is no time limit then there is no incentive to burn anything UNLESS it was made mandatory to an extent. If made permanent it would almost be akin to an achievement ingame (burn x wood, get z reward) which would be fine, but it would not really be an event of such whcih si what this proposal is aiming for. A limited time event for players to optionally partake in.

Intrinsic burns/sinks in the game are a topic I brought up many times and described here:
https://github.com/sunflower-land/sunflower-land/discussions/2977
https://github.com/sunflower-land/sunflower-land/discussions/2379

I would definitely love to have those, along with optional one-off time limited events on the side.

**Comment by rodolfoddias** [comment]
LOVE the idea. Miss the goblin wars so much! It's always great to have a community event to shake things up!

Regarding the requirements, I would consider setting the requirements based on the time frame, to avoid "pay to win" sentiment among the community. After all, it's a community event. 

Said that, I would set the requirements with a formula like: Time frame * number of times the player can mint the resource in a VERY GRINDY gameplay. Example: How many woods a player can farm taking down trees 16h per day non stop?

Players would probably still need to buy woods, but it would still be possible to grind their way to the prize. I also rather having many weak utility NFTs than few powerfull ones, since is a competition where the community join forces to win against the environment. 

I like the idea of having exclusive decorations for high volume donations! As a collector, I would do it for sure! It's something only true players would go for.

**Comment by rasheed19871** [comment]
I am all in for that I think wood is swarming so let's burn some wood

**Comment by bayselb** [comment]
Would definitely help with the over supply of wood. Would also like to see this with stone, iron and gold too. Anything that can keep the supply down and bring up the demand is a win in my opinion.
